cobalt/browser/application.cc

```diff
@@ -458,7 +458,12 @@ std::string GetMinLogLevelString() {
 }
 
 int StringToLogLevel(const std::string& log_level) {
-  if (log_level == "info") {
+  if (log_level == "verbose") {
+    // The lower the verbose level is, the more messages are logged.  Set it to
+    // a lower enough value to ensure that all known verbose messages are
+    // logged.
+    return logging::LOG_VERBOSE - 15;
```

Comment:
Is there documentation for the valid verbose levels?

Reply:
I didn't find document for it during my brief search.  It seems that the DVLOG() macro can accept arbitrary integer, and I saw code in Chromium using DVLOG(5) but not DVLOG(6).